tools/create_symbolic_links.py

```diff
@@ -24,6 +26,12 @@
     else:
         unreal_project_dirs = [ os.path.join(unreal_projects_dir, project) for project in os.listdir(unreal_projects_dir) ]
 
+    if args.unreal_plugins_dir:
```

Comment:
Don't you need else statements here? What happens if I don't specify args.unreal_plugins_dir?

Reply:
We don't because we are writing default values to this variable few lines above. If args.unreal_plugins_dir is input, we overwrite those default values.

Reply:
Ah ok. I prefer if-else for this kind of thing.

```
if args.unreal_plugins_dir:
    unreal_plugins_dir = args.unreal_plugins_dir
else:
    unreal_plugins_dir = whatever
```

Because it localizes the conditional logic to a very small part of the code. Otherwise, I need to scan through the code for a potentially unlimited number of lines to figure out the conditional logic.

Reply:
I agree, it's more clear this way. I see that you have made these changes. Thanks!